packages/react/ds-core-form/src/index.css

```diff
@@ -47,3 +54,9 @@
   --form-label-columns: 1 / -1;
   --form-payload-columns: 1 / -1;
 }
+
+/* Utility */
+
+.rtl {
+  direction: rtl;
+}
```

Comment:
In which situations would we want to override the text direction with a utility class? Is it just for demonstrative purposes in storybooks?

Reply:
If you define a `.rtl`, you should probably also define a `.ltr`